client/src/components/graph/overlays/graphOverlayLayer.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component, cloneElement } from "react";
+import { connect } from "react-redux";
+import { mat3 } from "gl-matrix";
+
+import styles from "../graph.css";
+
+export default
+@connect(state => ({
+  responsive: state.responsive
+}))
+class GraphOverlayLayer extends Component {
+  matrixToTransformString = m => {
+    /* 
+      Translates the gl-matrix mat3 to SVG matrix transform style
+
+          mat3         HTML Transform Function
+      [ a, b, 0       
+        c, d, 0,  =>  matrix(a, b, c, d, e, f) / matrix(sx, 0, 0, sy, tx, ty) / matrix(m[0] m[3] m[1] m[4] m[6] m[7])
+        e, f, 1 ]     
+    */
+    return `matrix(${m[0]} ${m[1]} ${m[3]} ${m[4]} ${m[6]} ${m[7]})`;
```

Comment:
comment does not match the code (indices 3 and 1 of the matrix are mismatched)

Reply:
fixed